payroll/serializers.py

```diff
@@ -18,10 +19,12 @@ class IndexSerializer(serializers.Serializer, ChartHelperMixin):
     salary_json = serializers.SerializerMethodField()
 
     def get_salary_count(self, data_year):
-        return Salary.objects.filter(vintage__standardized_file__reporting_year=data_year).count()
+        count = Salary.objects.filter(vintage__standardized_file__reporting_year=data_year).count()
+        return format_exact_number(count)
 
     def get_unit_count(self, data_year):
-        return Unit.objects.filter(vintage__standardized_file__reporting_year=data_year).count()
+        count = Unit.objects.filter(vintage__standardized_file__reporting_year=data_year).count()
+        return format_exact_number(count)
 
     def get_department_count(self, data_year):
```

Comment:
Aside: Do we use this anywhere in the template? If not, we can get rid of this method and the `department_count` attribute of this serializer class.